receiver/googlecloudspannerreceiver/config_test.go

```diff
@@ -27,6 +27,10 @@ import (
 	"go.opentelemetry.io/collector/receiver/scraperhelper"
 )
 
+const (
+	cardinalityOverallLimit = 200000
```

Comment:
```suggestion
cardinalityLimit = 200000
```

keeping the naming simple :)

Reply:
Done.